src/app/guards/auth.guard.ts

```diff
@@ -74,6 +74,12 @@ export class AuthGuard implements CanActivate {
       return this.router.parseUrl('/');
     }
 
+    if (
+      !this.keycloakService.isAllowed('faq') &&
+      state.url === '/faq'
+      ) {
```

Comment:
Prefer this to be a single line:

```
 if (!this.keycloakService.isAllowed('faq') && state.url === '/faq') {
```

Reply:
Fixed in [Commit](https://github.com/bcgov/parks-reso-admin/pull/338/commits/aca28aca017b37975d9ca5d3b95ad282c013b829)